etc/org.opencastproject.userdirectory.studip-default.cfg.template

```diff
@@ -0,0 +1,19 @@
+# Studip UserDirectoryProvider configuration
+
+# This is an an optional service which is not enabled by default. To enable it,
+# edit etc/org.apache.karaf.features.cfg and add opencast-studip to the featuresBoot option.
+
+# The organization for this provider
+org.opencastproject.userdirectory.studip.org=mh_default_org
+
+# The URL and token for the Studip REST webservice
+org.opencastproject.userdirectory.studip.url=https://my-studip.de/studip5/plugins.php/opencast/api/
+org.opencastproject.userdirectory.studip.token=mytoken1234abcdef
```

Comment:
These should be unset I'd think?  Take a look at the LDAP config file, equivalent keys are present, but unset.

Reply:
I was going off of the Moodle and Sakai config files, which also have those set. 

I think it would be a good idea to at least keep the url in some shape or form, so that the admin doesn't need to look up which endpoint this should target. If you want, I'll unset the url and token and make mention of the endpoint in a comment.

Reply:
Fair enough.  It's not going to cause significant confusion IMO since it *has* to be reset for it to work at all :)